src/app/toolbar/toolbar.component.html

```diff
@@ -5,6 +5,7 @@
     </mat-icon>
   </button>
   <span class="title" fxFlex> RxJS Docs </span>
+  <span class="title" fxFlex> WARNING: This is BETA site </span>
```

Comment:
@necoline can we have a - For the current RxJS docs, visit http://reactivex.io/rxjs under that?